addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -52,7 +52,7 @@
 
 <template id="s_newsletter_subscribe_popup" name="Newsletter Popup">
     <div class="o_newsletter_popup" data-list-id="0">
-        <div class="alert alert-warning css_non_editable_mode_hidden o_not_editable" role="alert">
+        <div class="alert alert-warning css_non_editable_mode_hidden o_editable" role="alert">
```

Comment:
i'm not sure why this is needed, doesn't the alert show just by maintaining CTRL when clicking?

Reply:
it's not necessary to trigger the click, but it makes the banner editable

Reply:
the newsletter is already editable

> ![2020-07-02-112034_1920x1080_scrot](https://user-images.githubusercontent.com/9977887/86340891-de893f00-bc55-11ea-98f7-29d9e58b2531.png)

If you mean editing this part:

![2020-07-02-112154_618x63_scrot](https://user-images.githubusercontent.com/9977887/86341002-01b3ee80-bc56-11ea-9205-6e9541094517.png)

it's part of the edition interface, there is no reason to edit it.